data/csv.go

```diff
@@ -0,0 +1,87 @@
+package data
+
+import (
+	"encoding/csv"
+	"errors"
+	"go-bootcamp/model"
+	"io"
+	"os"
+	"strconv"
+)
+
+type Csv struct {
+	index       map[int]model.Pokemon
+	initialized bool
+}
+
+func (storage Csv) All() ([]model.Pokemon, error) {
```

Comment:
For all exported stuff (functions, struct, variable, etc.) Please add comment description like:

// Example - Description here...